features/step_definitions/helper_services.rb

```diff
@@ -500,9 +500,9 @@
 Given /^I have a iSCSI setup in the environment$/ do
   ensure_admin_tagged
 
-  _project = project("iscsi-target", switch: false)
+  _project = project("openshift-iscsi-target", switch: false)
```

Comment:
Perhaps use `openshift-qe-iscsi-target` ? Just in case the project name is used by OCP

Reply:
That makes sense even OCP doesn't have it now.
Adding `qe` or `qe-test`.

Reply:
Thanks for the suggestion! Fixed by using another solution.